3p-ecosystem/src/main/java/com/google/homesampleapp/screens/commissionable/CommissionableFragment.kt

```diff
@@ -59,8 +78,8 @@ class CommissionableFragment : Fragment() {
 
     lifecycleScope.launch {
       lifecycle.repeatOnLifecycle(Lifecycle.State.RESUMED) {
-        viewModel.beaconsFlow.distinctUntilChanged().collect { beacon ->
```

Comment:
Since the beacons are observed as state, This flow may not need to be collected.

Reply:
Agree, removed.